packages/ra-core/src/util/fetch.js

```diff
@@ -54,7 +54,7 @@ const isValidObject = value => {
     }
 
     const isArray = Array.isArray(value);
-    const isBuffer = Buffer.isBuffer(value);
+    const isBuffer = Buffer && Buffer.isBuffer(value);
```

Comment:
So we won't know if it's a buffer on Windows ?

Reply:
`Buffer` is a Node object - it does not exist natively in browsers. It's emulated by the `buffer` npm module, which creates a global `Buffer` object. This `buffer` module is required by webpack. So if you use webpack, you don't have the bug. I think the bug appears not if you're on Windows, but if you use another bundler (like rollup). My fix patches that.

But the heart of the problem is that the value cannot be a Buffer since it doesn't make sense for browsers. This test was added in #834, and I validated the PR, but I don't understand this Buffer test anymore.

Reply:
Yes, you are right: it's not Windows problem. But neutrino using Webpack too. May be some CRA module setup global Buffer object unlike neutrino.